resources/services/cloudwatchlogs/gen.hcl

```diff
@@ -0,0 +1,41 @@
+resource "aws" "cloudwatchlogs" "log_groups" {
+  path = "github.com/aws/aws-sdk-go-v2/service/cloudwatchlogs/types.LogGroup"
+  ignoreError "IgnoreAccessDenied" {
+    path = "github.com/cloudquery/cq-provider-aws/client.IgnoreAccessDeniedServiceDisabled"
+  }
+  deleteFilter "AccountRegionFilter" {
+    path = "github.com/cloudquery/cq-provider-aws/client.DeleteAccountRegionFilter"
+  }
+  multiplex "AwsAccountRegion" {
+    path   = "github.com/cloudquery/cq-provider-aws/client.ServiceAccountRegionMultiplexer"
+    params = ["logs"]
+  }
+
+  userDefinedColumn "account_id" {
+    description = "The AWS Account ID of the resource."
+    type        = "string"
+    resolver "resolveAWSAccount" {
+      path = "github.com/cloudquery/cq-provider-aws/client.ResolveAWSAccount"
+    }
+  }
+
+  userDefinedColumn "region" {
+    type        = "string"
+    description = "The AWS Region of the resource."
+    resolver "resolveAWSRegion" {
+      path = "github.com/cloudquery/cq-provider-aws/client.ResolveAWSRegion"
+    }
+  }
+  ignore_columns_in_tests = ["kms_key_id","retention_in_days"]
+
+  options {
+    primary_keys = ["arn"]
+  }
+
+  column "tags" {
+    type = "json"
+    resolver "resolveTags" {
+      path = "github.com/cloudquery/cq-provider-aws/client.ResolveTags"
+    }
+  }
```

Comment:
Because `tags` aren't included in the describe response you have to have a column resolver that actually makes the `list-tag-log-groups` call

Here is an example of how another resource that follows the same pattern:
https://github.com/cloudquery/cq-provider-aws/blob/26a83395ed007ed8a888aea45b6037b25e0af38f/resources/services/athena/data_catalogs.go#L241-L263

It also means you are going to have add the function to the CloudwatchlogClient in `client/services.go`

Reply:
Here is the associated `cq-gen` config: 
```
  userDefinedColumn "tags" {
    type              = "json"
    generate_resolver = true
  }
```

Reply:
Added! Thank you, that was very helpful!